src/views/dialogs/DepositDialog2/DepositStatus.tsx

```diff
@@ -70,21 +56,14 @@ export const DepositStatus = ({ txHash, chainId, onClose }: DepositStatusProps)
         </div>
       </div>
       <div tw="flex items-center justify-between self-stretch">
-        <div tw="text-color-text-0">{depositSuccess ? 'Available balance' : 'Your deposit'}</div>
+        <div tw="text-color-text-0">Your deposit</div>
         <div tw="flex items-center gap-0.125">
-          {depositSuccess ? (
-            <>
-              <ArrowRightIcon tw="text-green" />
```

Comment:
so the subaccount sweep doesnt sync up to the freeCollateral loading again, so this screen always flashes the old collateral value before the new one shows up. instead of handling this case ima just remove this from the spec to simplify things ...